### DISCUSSION TITLE
Android 14

### DISCUSSION THREAD

**Discussion by sensorial7** [fcaronte/KingInstaller#15] [Q&A]
Hello, There is a problem with the Android 14 platform, after installing Fermata Auto and Control version 1.9.5 via Kinginstaller 1.4, there are no fermata icons in the AndroidAuto service. Everything worked on the previous version of Android 13. When can we expect the new version of Fermata dedicated to Android 14

**Comment by fcaronte** [comment]
> Hello, There is a problem with the Android 14 platform, after installing Fermata Auto and Control version 1.9.5 via Kinginstaller 1.4, there are no fermata icons in the AndroidAuto service. Everything worked on the previous version of Android 13. When can we expect the new version of Fermata dedicated to Android 14

Is not only a Fermata problem, but all custom app stop working with A14
